### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Daniil-Pankieiev**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -2,6 +2,48 @@
...
+        ("Keanu", "Reeves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    genres = ["Western", "Action", "Drama"]
+
+    for item_first, item_second in actors:
+        Actor.objects.create(
```
Comment (Familenko) [inline main.py:20]: You're very clever, you wrote down the first and last names separately)). But please label the variables unpacked from the tuple according to their contents

File: main.py [hunk 2]
```diff
@@ -2,6 +2,48 @@
...
+            first_name=item_first,
+            last_name=item_second)
+
+    for item in genres:
+        Genre.objects.create(name=item)
```
Comment (Familenko) [inline main.py:25]: same issue here

File: main.py [hunk 3]
```diff
@@ -2,6 +2,48 @@
...
+    ).update(last_name="Clooney")
+
+    Actor.objects.filter(
+        first_name="Kianu",
+        last_name="Reaves"
+    ).update(first_name="Keanu", last_name="Reeves")
+
+    Genre.objects.filter(
+        name="Action",
+    ).delete()
```
Comment (Familenko) [inline main.py:41]: please use same style in all code:

in this case you make a block

    Genre.objects.filter(
        name="Action",
    ).delete()

but in same situation you make one line
```
Genre.objects.filter(name="Dramma",).update(name="Drama")
```

**Review by Familenko — CHANGES_REQUESTED** [review]

**Review by fsocie7y — APPROVED** [review]
GJ 👍

**Pull request closed without merging**